charts/SAML/values.yaml

```diff
@@ -0,0 +1,42 @@
+deployment:
+  imagePullSecrets: []
+  image: docker.io/insightfinderinc/saml:latest
+  resources:
+    requests:
+        cpu: 100m
+        memory: 300Mi
+
+ingress:
+  ingressClassName: nginx
+  # Host name for ingress/saml server
+  host:
+  tls:
+    enabled: true
+    secretName: ingress-tls
+
+insightfinder:
+  # The URL of the InsightFinder server
+  serverUrl: https://app.insightfinder.com
+  # The URL of the this SAML server
+  samlUrl:
+  # saml application acts as service provider, it needs key and cert to make requests.
+  # key and cert need to be in same secret, with 'key' and 'cert' as the secret keys
+  spSecretName: sp-secret
+
+saml:
+  idp:
+    # List of Identity Providers
+    teleport:
+      entityid: https://xxx.teleport.sh/enterprise/saml-idp/metadata
+      # use firstname-key to extract first name info of the user from response of idp
+      firstnamekey: firstname-field-in-the-principal
+      lastnamekey: lastname-field-in-the-principal
+      # use email-key to extract email info of the user from response of idp
+      emailkey: email-field-in-the-principal
+      # (Optional) The URL of the Identity Provider's SAML 2.0 SSO service
+      singlesignonurl: https://xxx.teleport.sh/enterprise/saml-idp/sso
+      # True if providing certificate for idp in the secrets
+      idp_cert_provided: false
```

Comment:
Can we put the certificates data in a single secret / configmap?
You can create secret/cm named:  `if-saml-idp-xxx-cert`